junit5-testing/src/test/java/org/jsmart/zerocode/jupiter/extension/ParallelLoadExtensionTest.java

```diff
@@ -0,0 +1,71 @@
+package org.jsmart.zerocode.jupiter.extension;
+
+import java.lang.reflect.Method;
+import org.jsmart.zerocode.core.domain.LoadWith;
+import org.jsmart.zerocode.core.domain.TestMapping;
+import org.jsmart.zerocode.core.domain.TestMappings;
+import org.jsmart.zerocode.tests.jupiter.JUnit5Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ParallelLoadExtensionTest {
```

Comment:
👍 
Pls add more examples into the HelloWorld Load repo.

Reply:
will do.
Already a PR with few handy examples to unblock the user are there!
PR link: https://github.com/authorjapps/performance-tests/pull/3/files